.github/file-paths.yaml

```diff
@@ -13,7 +13,6 @@ shared_specs: &shared_specs
 
 frontend_sources: &frontend_sources
   - *shared_sources
-  - "frontend/**"
   - "enterprise/frontend/**"
   - "frontend/!(test)/**"
```

Comment:
It seems we cannot have both `frontend/**` and then later a narrower scope that would exclude the test folder. In fact, `frontend/(!test)/**` alone seems to be the exact glob that we need to compare changed files against.